src/Nuget.PackageIndex/LocalPackageIndexBuilder.cs

```diff
@@ -136,7 +136,13 @@ public Task<LocalPackageIndexBuilderResult> BuildAsync(bool shouldClean = false,
                     {
                         if (cancellationToken != null && cancellationToken.IsCancellationRequested)
                         {
-                            return new LocalPackageIndexBuilderResult { Success = false, TimeElapsed = stopWatch.Elapsed }; ;
+                            return new LocalPackageIndexBuilderResult { Success = false, TimeElapsed = stopWatch.Elapsed };
```

Comment:
That's not what mutable means. It's a question about using properties vs constructor